### ISSUE TITLE
"no container" should be a container itself, it's missing from the menu

### ISSUE THREAD

**Issue by g-i-o-r-g-i-o** [mozilla/multi-account-containers#2701]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

Click on the container button.

There is no 
"no container" option

This is necessary because, when you use a container and you click a link, all the websites are opened in it. I should be able to decide that if I open youtube.com, wherever the container I'm in, that link should be opened in the "no container" if I decided explicitly to do so.

### Actual behavior

click on the address bar button:

```
"Always open this site in ..."

```

the "no container" option is missing

### Expected behavior

I should have

`"---> no container"`

**Comment by Joshua-Shepherd** [comment]
The main issue is you've added this feature, and once you start using it.. You can't reset it or stop it.
![CleanShot 2024-12-16 at 19 39](https://github.com/user-attachments/assets/d5e69d26-9391-4cec-a6c4-2975e853bb51)

**Comment by kurahaupo** [comment]
I agree that it would be nice if that dialogue offered more options than just "current container" and "assigned container", but it is helpful to highlight those two options first.

 indeed it would be preferable to select from possible containers.

But that said, if intending to open a link in a specific container (including the "no" container), it's quicker just to right click and use "open link in container" and skip the dialog entirely.

Maybe a shorter shortcut could be nice, perhaps hook ctrl-click and shift-click (which normally open new tabs and new windows) to ask about the container and _then_ open the new tab or window.

(And before anyone argues about "not wanting a new tab", keep in mind that you _always_ get a new tab, just that sometimes the old tab is automatically closed. You can see this in the fact that the "back" button does nothing, and re-open tab does indeed restore the previous page.)

**Comment by loganrosen** [comment]
This is a duplicate of #503.
